web-ui/src/app/components/filters/int/int.component.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, OnInit} from '@angular/core';
+import {FilterComponent} from "../filter/filter.component";
+import {Filter} from '../filters.component';
+import {FilterRangeValue} from "../../../services/results.service";
```

Comment:
In Visual Studio Code you can use `ctrl + shift + i` to automatically format your code, it should automatically respect settings like tab width then and places spaces around `{}` etc.